.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+updates:
+  - package-ecosystem: "yarn"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    groups:
+      cannon:
```

Comment:
experimenting here. Dependabot will probably try to update all deps anyway. Need to adjust it as we go